src/lib389/lib389/cli_conf/backend.py

```diff
@@ -94,7 +95,7 @@ def _search_backend_dn(inst, be_name):
         cn = ensure_str(be.get_attr_val('cn')).lower()
         suffix = ensure_str(be.get_attr_val('nsslapd-suffix')).lower()
         del_be_name = be_name.lower()
-        if cn == del_be_name or suffix == del_be_name:
+        if cn == del_be_name or str2dn(suffix) == str2dn(del_be_name):
```

Comment:
Should really use "===" instead of "=="

Reply:
Fix me if I'm wrong, but I think there's no such thing in Python...

Reply:
Ummm, yeah you're right haha